src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/SwaggerGenerator.cs

```diff
@@ -112,6 +112,11 @@ public OpenApiDocument GetSwagger(string documentName, string host = null, strin
             }
         }
 
+        public string[] GetDocumentNames()
+        {
+            return _options.SwaggerDocs.Keys.ToArray();
+        }
```

Comment:
This can be simplified to an expression:

```suggestion
        public string[] GetDocumentNames() => _options.SwaggerDocs.Keys.ToArray();
```

Reply:
Fixed